frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -74,12 +75,17 @@ export const reloadCard = createThunkAction(RELOAD_CARD, () => {
       Questions.actions.fetch({ id: outdatedQuestion.id() }, { reload: true }),
     );
     const card = Questions.HACK_getObjectFromAction(action);
+    const question = new Question(
+      card,
+      getMetadata(getState()),
+      outdatedQuestion.parameters(),
+    );
```

Comment:
I was assuming this was covered in the [saved question tests suite](https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/question/saved.cy.spec.js). I can now see that that test suite does not have tests with parameters set, so that a wrong assumption.

OTOH, the behavior around parameters and reverting questions (which is the only place where `RELOAD_CARD` get's triggered AFAICT), seems broken on `master` too: the query gets run without parameters even if a parameter is set.
I can try and fix that (kind of unrelated) bug in this PR too, or create a separate issue and PR for that.

Reply:
Let's file a separate issue for it :+1: